Applications/Spire/Include/Spire/KeyBindings/KeyBindingsModel.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef SPIRE_KEY_BINDINGS_MODEL_HPP
+#define SPIRE_KEY_BINDINGS_MODEL_HPP
+#include "Spire/KeyBindings/OrderTask.hpp"
+#include "Spire/Ui/ComboBox.hpp"
+
+namespace Spire {
+
+  /** Defines the interface representing key bindings */
+  class KeyBindingsModel {
+    public:
+      virtual ~KeyBindingsModel() = default;
+
+      /** Returns the key bindings of the order tasks. */
+      virtual std::shared_ptr<ListModel<OrderTask>> get_order_tasks() = 0;
```

Comment:
Determine what aspects of this model are read-only/const and mark the signature accordingly.